packages/amplify_datastore/example/ios/unit_tests/AmplifyModelSchemaUnitTests.swift

```diff
@@ -19,46 +19,95 @@ import Amplify
 @testable import amplify_datastore
 
 class AmplifyModelSchemaUnitTests: XCTestCase {
-    
-    let schemasMap: [String: Any] = try! readJsonMap(filePath: "model_schema_maps")
+
+    let modelSchemaMap: [String: Any] = try! readJsonMap(filePath: "model_schema_maps")
+    let customTypeSchemaMap: [String: Any] = try! readJsonMap(filePath: "custom_type_schema_maps")
+    let customTypeSchemasRegistry = FlutterSchemaRegistry()
+
+    override func setUpWithError() throws {
```

Comment:
Can we test both model schemas and custom type schemas?

Reply:
Both model and custom type schemas are included in this test suite. Custom type schemas are added within `setUpWithError` by deserializing serialized custom type schemas.